src/Components/App/App.tsx

```diff
@@ -1,11 +1,8 @@
 import React, { useEffect, useState } from 'react';
-// import { Route } from 
 import './App.css';
 import Nav from '../../Components/Nav/Nav'
 import { User } from '../../interfaces'
-import { user1 } from '../../mockData'
-import { dummyUser } from '../../interfaces'
-import { dummyMed } from '../../interfaces'
+import { QueryRx } from '../../interfaces'
```

Comment:
Should be able to import multiple interfaces if from the same file:

`import { User, QueryRx } from '../../interfaces'`